ClrMD.Extensions/ClrMDSession.cs

```diff
@@ -33,25 +33,19 @@ public class ClrMDSession : IDisposable
         public bool IsReferenceMappingCreated { get; private set; }
 
         internal ClrType ErrorType { get; private set; }
-
+        
         private ClrMDSession(DataTarget target, string dacFile)
         {
             ClrMDSession.Detach();
 
-            if (target.ClrVersions.Count == 0)
+            if (target.ClrVersions.Count() == 0)
```

Comment:
Depending of the underlying type, `Count()` could unnecessary iterate all elements. Should use `.Any()` instead of `Count() == 0`.